cmd/create/machinepool/helper.go

```diff
@@ -79,3 +82,21 @@ func getSubnetOptions(r *rosa.Runtime, cluster *cmv1.Cluster) ([]string, error)
 
 	return subnetOptions, nil
 }
+
+func getSecurityGroupsOption(r *rosa.Runtime, cmd *cobra.Command, cluster *cmv1.Cluster) ([]string, error) {
+	if len(cluster.AWS().SubnetIDs()) == 0 {
+		return []string{}, fmt.Errorf("Expected cluster's subnets to contain subnets IDs, but got an empty list")
+	}
+
+	availableSubnets, err := r.AWSClient.GetVPCSubnets(cluster.AWS().SubnetIDs()[0])
+	if err != nil {
+		return []string{}, fmt.Errorf("Failed to retrieve available subnets: %v", err)
+	}
+	firstSubnet := availableSubnets[0]
+	vpcId := awssdk.StringValue(firstSubnet.VpcId)
+	if vpcId == "" {
+		return []string{}, fmt.Errorf("Unexpected situation a VPC ID should have been selected based on chosen subnets")
+	}
```

Comment:
Would it be worth pulling this out to its own small function to make it easy to test without mocking.